md/bdm-access-control.md

```diff
@@ -314,3 +315,163 @@ Deploy your access control file.
 Access to your previous data is now controlled by the BDM Access Control file you just deployed. An order picker can't see the customer and the price of a product anymore. A Sales can't see the address of a customer anymore.  
 Connect onto the portal as a user with the profile *Order picker*. Go to your application: the customer data and the order price are always empty.
 Connect now as a user with the profile *Sales*. On your application, the customer data are displayed except the address. The price of the product is now available.
+
+
+### III - Instances protection
+
+Instance protection is not supported as a part of the BDM Access Control feature. However it is still possible to solve most use cases using the [rest-api authorizations](rest-api-authorization.md).
+Following is a step by step guide on how to realize one such (simple) use case:
+
+::: info
+The method shown in the example below is not a new 7.7 feature, it is present in the Community version since 7.0 ( although the location of the files and the push/pull procedure changed in 7.6.0).
```

Comment:
The example below accounts for a specific way to use a method introduced in Bonita 7.0, and updated in Bonita 7.6.